yamilovs/sms-bundle/1.0/post-install.txt

```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                                           </>
+<bg=blue;fg=white> Getting started using </><bg=blue;fg=white;options=bold>yamilovs/sms-bundle</><bg=blue;fg=white> </>
```

Comment:
We are trying to remove the use of too many colors. Could you please remove this?

Reply:
Also, the extra markup you are using `<bg=blue;fg=white;options=bold>` does not do anything... 

![screen shot 2018-04-03 at 11 02 15](https://user-images.githubusercontent.com/1275206/38239988-95dfc700-372e-11e8-8089-e123aca6c1e5.png)


Just keep the markup in the beginning of the line and Im very happy with this PR.

Reply:
Done at https://github.com/symfony/recipes-contrib/pull/253/commits/645011b994a2ab60f9b7bb48a2f6f6bb7fdc2119